tools/cloud-build/daily-tests/tests/ml-a3-highgpu-slurm-cluster.yml

```diff
@@ -41,6 +41,4 @@ cli_deployment_vars:
   server_ip_homefs: "{{ nfs_ip }}"
   remote_mount_homefs: "{{ remote_mount_homefs }}"
   slurm_cluster_name: "{{ slurm_cluster_name }}"
-  disable_login_public_ips: "false"
```

Comment:
why we remove this ?

Reply:
The parameters in latest v6 modules are called `enable_login_public_ips` and they are already set to `true`.